src/app/tests/suites/certification/Test_TC_TSTAT_2_1.yaml

```diff
@@ -139,56 +138,56 @@ tests:
               maxValue: -25
 
     - label: "Reads optional attributes from DUT: OccupiedCoolingSetpoint"
-      PICS: TSTAT.S.A0011
+      PICS: TSTAT.S.F01
       command: "readAttribute"
       attribute: "OccupiedCoolingSetpoint"
       response:
           constraints:
               type: int16
-              minValue: -27315
-              maxValue: 32767
+              minValue: 1600
+              maxValue: 3200
```

Comment:
Why?  What if this thermostat has an AbsMinCoolingSetPoint of -200 and an OccupiedCoolingSetpoint of 0?  Similar for other places in this file where minValue/maxValue got changed.

Reply:
These are the defaults for ABS min and ABS max set point limits. It would be very unusual for a home thermostat to be set outside of those ranges.  If you feel it is critical that they be to the outside limits of the default ABS values then I can change them. That will have an impact though on the test steps that try to set the setpoints to the limit and outside the limits. Not sure that we can handle this currently with the TH.  

I can create an issue to address this situation in the future, but  this condition seems well outside of SVE scope.

Reply:
https://github.com/project-chip/connectedhomeip/issues/21086

Reply:
I guess my question is this: should a thermostat that has different min/max limits fail certification?  Because that's the current situation.  And yes, I agree that it's not just the minValue/maxValue here but the other bits as well....